docs/rest-api.yml

```diff
@@ -425,8 +425,9 @@ paths:
         '200':
           description: The user or service identified by the API token
           schema:
-            $ref: '#/definitions/User'
-            $ref: '#/definitions/Service'
+            $ref: 
+              - '#/definitions/User'
+              - $'#/definitions/Service'
```

Comment:
Yep. Good catch. I was having odd errors with RTD (npm build errors) all day so I may let this sit until morning.